pallets/common/src/lib.rs

```diff
@@ -14,8 +14,45 @@
 // You should have received a copy of the GNU General Public License
 // along with Unique Network. If not, see <http://www.gnu.org/licenses/>.
 
+//! # Common pallet
+//!
+//! The Common pallet provides functionality for handling collections.
+//!
+//! ## Overview
+//!
+//! The Common pallet provides functions for:
+//!
+//! - Setting and approving collection soponsor.
+//! - Get\set\delete allow list.
+//! - Get\set\delete collection properties.
+//! - Get\set\delete collection property permissions.
+//! - Get\set\delete token property permissions.
+//! - Get\set\delete collection administrators.
+//! - Checking access permissions.
+//! - Provides an interface for common collection operations for different collection types.
+//! - Provides dispatching for implementations of common collection operations, see [dispatch] module.
+//! - Provides functionality of collection into evm, see [erc] and [eth] module.
```

Comment:
Could you also remove these three lines from here, since they're redundant now? Also, could you put that text you've moved out _into_ overview, before "The Common pallet provides functions for: [...]", and put `The Common pallet provides functionality for handling collections.` back as a one-line description?